shared/OpenTelemetry/src/sdk/OTelLogRecord.ts

```diff
@@ -33,13 +35,28 @@ export function createLogRecord(
         context
     } = logRecord;
 
-    const logAttributes = attributes || {};
+    const hasAttributeContainer = isAttributeContainer(attributes as any);
+    const logAttributes = (!hasAttributeContainer && attributes) ? attributes : {};
     const now = Date.now();
     const hrTime = timeInputToHrTime(timestamp || now);
     const hrTimeObserved = timeInputToHrTime(observedTimestamp || now);
     const resource = sharedState.resource;
     const logRecordLimits: Required<IOTelLogRecordLimits> = sharedState.logRecordLimits;
     const handlers: IOTelErrorHandlers = {};
+    const attributeContainer = createAttributeContainer<any>(
+        {
+            traceCfg: {
```

Comment:
The config should be injected from the main config and not applied here as it's own version (non-user configurable)